.gitignore

```diff
@@ -12,9 +12,10 @@
 [Aa]rtifacts/
 [Dd]ebug/
 [Rr]elease/
-x64/
 [Bb]in/
 [Oo]bj/
+[Pp]ackages/
```

Comment:
Not sure this should be necessary.  Arcade uses ".packages".  But it doesn't hurt anything.